pkg/engine/imageVerify.go

```diff
@@ -468,7 +474,7 @@ func (iv *imageVerifier) verifyAttestorSet(
 				cosignResp, entryError = iv.verifyAttestorSet(ctx, *nestedAttestorSet, imageVerify, imageInfo, attestorPath, predicateType)
 			}
 		} else {
-			opts, subPath := iv.buildOptionsAndPath(a, imageVerify, image, kyvernov1.Attestation{PredicateType: predicateType})
+			opts, subPath := iv.buildOptionsAndPath(a, imageVerify, image, nil)
```

Comment:
I just noticed `predicateType` was removed here, while `verifyAttestorSet` is also used in attestations checks given the specific predicateType `verifyImages.attestations.predicateType`.